examples/plugins/ethdoc/webpack.common.js

```diff
@@ -0,0 +1,20 @@
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+const path = require('path')
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.ts?$/,
+        exclude: [/node_modules/],
+        loader: "ts-loader",
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.ts', '.js'],
+    plugins: [
+      new TsconfigPathsPlugin({ configFile: path.resolve(__dirname, './tsconfig.json') }),
```

Comment:
I think you want to get the tsconfig.json from the root here instead : `path.resolve(__dirname, '../../../tsconfig.json')`